.circleci/config.yml

```diff
@@ -3,18 +3,23 @@
 # Check https://circleci.com/docs/2.0/language-go/ for more details
 version: 2
 jobs:
-  build:
+   build:
     docker:
       - image: circleci/golang:1.13
-        environment: GO111MODULE=on
+        environment: 
+          - GO111MODULE=on
+          - CGO_ENABLED=0
+
     working_directory: /go/src/github.com/kiegroup/kogito-cloud-operator
     steps:
```

Comment:
Maybe not now, but on a different JIRA we could reorganize this CI.

Reply:
Okay, I'll revert this change